Content.Client/Cargo/BUI/CargoOrderConsoleBoundUserInterface.cs

```diff
@@ -68,7 +68,7 @@ protected override void Open()
 
             _menu.OnItemSelected += (args) =>
             {
-                if (args.Button.Parent is not CargoProductRow row)
+                if (args.Button.Parent?.Parent is not CargoProductRow row)
```

Comment:
Is there any better way of doing this? It feels like a bad code smell to have to rely on the linked-lists of ui nodes.

Reply:
Let's fix all this without me, if you need, and I'll stand aside. I haven't changed anything in principle, so you can assume that I didn't touch this file.